.github/workflows/release.yml

```diff
@@ -0,0 +1,119 @@
+# - Comments on the the release PR pointing to the GitHub release URL.
+name: Release
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - 'release/**'
+
+jobs:
+  tag:
+    name: Create tag
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    outputs:
+      version_tag: v${{ steps.version.outputs.release_version_short }}
+    steps:
+      - name: Create GitHub App Token
+        uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ vars.RELEASE_BOT_APP_ID }}
+          private-key: ${{ secrets.RELEASE_BOT_PRIVATE_KEY }}
+
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          token: ${{ steps.app-token.outputs.token }}
+          sparse-checkout: |
+            package.json
+          sparse-checkout-cone-mode: false
+
+      - name: Parse version
+        id: version
+        uses: ./.github/parse-version
+
+      - name: Tag release
+        run: |
+          git config --global user.name 'awana-release-bot[bot]'
+          git config --global user.email '${{ vars.RELEASE_BOT_USER_ID }}+awana-release-bot[bot]@users.noreply.github.com'
+          git tag ${{ steps.version.outputs.releaseShort }}
+          git push origin git tag ${{ steps.version.outputs.releaseShort }}
+
+  create_release:
+    name: Create release
+    needs: tag
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create GitHub App Token
+        uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ vars.RELEASE_BOT_APP_ID }}
+          private-key: ${{ secrets.RELEASE_BOT_PRIVATE_KEY }}
+
+      - name: Create release on GitHub
+        env:
+          GITHUB_TOKEN: ${{ steps.app-token.outputs.token }}
+          REPOSITORY: ${{ github.repository }}
+          TAG_NAME: ${{ needs.tag.outputs.version_tag }}
+        run: |
+          release_url=$(gh release create \
+            --repo "$REPOSITORY" \
+            --draft \
+            --verify-tag \
+            --generate-notes \
+            --title "$TAG_NAME" \
+            "$TAG_NAME" | jq '.url' --raw-output)
+
+          echo "release_url=$release_url" >> $GITHUB_OUTPUT
+
+  build:
+    name: Create builds
+    needs: tag
+    uses: ./.github/workflows/create-builds.yml
+    with:
+      app_type: 'production'
+      build_sha: ${{ github.sha }}
+    secrets: inherit
+
+  comment:
+    name: Comment
+    needs: [create_release, build]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create GitHub App Token
+        uses: actions/create-github-app-token@v1
+        id: app-token
+        with:
+          app-id: ${{ vars.RELEASE_BOT_APP_ID }}
+          private-key: ${{ secrets.RELEASE_BOT_PRIVATE_KEY }}
+
+      - name: Parse version
+        id: version
+        uses: ./.github/parse-version
```

Comment:
don't think you need this any more

Reply:
addressed via https://github.com/digidem/comapeo-desktop/pull/190/commits/99c1fd20d71abf2c61a1352a58c09620aeea8cc0